### ISSUE TITLE
Missing test and submit code buttons

### ISSUE THREAD

**Issue by K3tna** [rage/tmc-vscode#674]
I can't test or submit my code because the buttons are missing. I have everything installed properly and double checked everything. I used TMC on my work pc and everything worked fine, I could test my code and submit it. However when I tried to work on my pc the buttons never appeared. Everything is up to date, I even disabled unrelated extensions to check if they caused the problem, but it didn't help. I've been trying different approaches for few hours and nothing works. I also tried launching it in VSCodium to check if it helps. Other TMC functions work perfectly fine

![image](https://github.com/rage/tmc-vscode/assets/25373825/7e21cad4-e1ca-490f-b750-db8372abfdd8)

**Comment by Heliozoa** [comment]
Can you access the functions from the command palette? (Press Ctrl+Shift+P, type in `TestMyCode` and you should see a list of commands offered by the plugin)

We had a similar report before but unfortunately we weren't able to reproduce it...

**Comment by K3tna** [comment]
Yes, after posting this issue I found the commands and finally was able to submit and test. But it's still weird why buttons are gone. For now I just made my own keyboard shortcuts for easier access.

**Comment by flavio-b-74** [comment]
I have the same issue... the buttons are gone, I'm still able to run the commands via command palette

**Comment by flavio-b-74** [comment]
Ok, I found how ro restore buttons: TMC button (on the left) -> My Courses -> Programming with Python 2023 -> Open workspace -> Do you want to open TMC workspace and close the current one? YES.
Then, I found the button again
